utils/components-utils/src/main/scala/pl/touk/nussknacker/engine/util/parameters/TestingParametersSupport.scala

```diff
@@ -10,20 +10,39 @@ object TestingParametersSupport {
 
   // Used to un-flat map with concat name eg. { a.b -> _ } => { a -> { b -> _ } }. Reverse kinda joinWithDelimiter
   def unflattenParameters(flatMap: Map[ParameterName, AnyRef]): Map[String, AnyRef] = {
+    unflattenParametersInternal(flatMap.map { case (ParameterName(name), value) => name -> value })
```

Comment:
It probably was me who named it `flatMap` but now I think that it is unfortunate name for scala variable :face_in_clouds: Maybe `flatParameters` / `flatParams` or something like this?

Reply:
You're right. It's a better name.